docs/messaging/messaging-spans.md

```diff
@@ -280,12 +275,13 @@ These attributes should be set to the broker to which the message is sent/from w
 
 ### Attribute namespaces
```

Comment:
I'm seeing such a namespace list for the first time, do you think it's actually needed? If so, it should be put before the tables IMHO.

Reply:
I personally think it's nice to have an overview like this. Agree with maybe moving it to before the table!

Reply:
I'm confident the table can be moved, but I'd like to keep it out of this PR, to avoid obfuscating the diff.